lib/wallet/records.js

```diff
@@ -227,6 +227,63 @@ class BlockMeta {
   }
 }
 
+/**
+ * TX Count
```

Comment:
Could you add a comment that describes this class? It looks pretty simple but could still use a description

Reply:
Can add brief description. As note many of these records lack a description.